sunbird-rc-plugin/src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@ mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.field-details={{"id":"pol
 mosip.esignet.authenticator.sunbird-rc.auth-factor.kbi.registry-search-url=https://registry.dev1.mosip.net/api/v1/Insurance/search
 mosip.esignet.authenticator.sunbird-rc.kbi.entity-id-field=osid
 
-## The mapping where the key represents the OIDC claim, and the value represents the corresponding claim from the Sunbird registry.
+## The mapping where the key represents the corresponding claim from the Sunbird registry, and the value represents the OIDC claim.
 mosip.esignet.authenticator.sunbird-rc.identity-openid-claims-mapping={"fullName":"fullName","name":"fullName","email":"email","phone":"mobile","gender":"gender","dateOfBirth":"dob","birthdate":"dob"}
```

Comment:
Based on the comment, key-value defined in the map is incorrect. Kindly refer this section to get the list of standard OIDC claims https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims